### PR TITLE
Feature/a11y

### PR DESCRIPTION
**Pull request by afewyards**

### CODE REVIEW

File: test/paper-date-picker.html [hunk 1]
```diff
@@ -47,22 +60,276 @@
...
+              calendar = element.$$('paper-calendar');
+              setTimeout(beforeDone, 0);
+            });
+          });
+
+          it('should expect the current month to be not null', function () {
+            expect(currentMonth).to.be.not.null;
+          });
+
+          it('should expect the current month to be null', function () {
```
Comment (Ex-iT) [inline test/paper-date-picker.html]: Description doesn't match the actual test.

File: test/paper-date-picker.html [hunk 2]
```diff
@@ -47,22 +60,276 @@
...
+
+        // =====================================================================
+        // TODO:
+        // =====================================================================
+        // Changes in aria states, identified here, as well as focus, are clearly styled to show the user where their point of regard is and what days are selected.
+        // when using the keyboard to navigate the datepicker
+        //  tabs to the months
+        //  current date
+        //  cancel
+        //  ok buttuns
```
Comment (Ex-iT) [inline test/paper-date-picker.html]: typo: ok buttons

File: test/paper-date-picker.html (hunk 1 above)
Comment (afewyards) [inline test/paper-date-picker.html]: done

File: paper-calendar.html [hunk 3]
```diff
@@ -681,14 +689,14 @@
...
-        selected.style.flex = '';
         window.requestAnimationFrame(function() {
           if (w > 0 && w < h) {
             selected.style.height = w + 'px';
           }
           else if (h > 0) {
-            selected.style.width = h + 'px';
+            selected.style.width = (h / w) * 100 + '%';
           }
+//          window.setTimeout(function() {selected.focus();}, 200);
```
Comment (afewyards) [inline paper-calendar.html]: done

**Pull request merged**